test/service/test_request_validation.py

```diff
@@ -153,6 +153,20 @@ def test_valid_sort_facet_but_bad_filter_facet_of_file(self):
         self.assertEqual(400, response.status_code, response.json())
         self.assertEqual(self.filter_facet_message, response.json())
 
+    def test_single_entity_error_responses(self):
+        self._index_canned_bundle(("f79257a7-dfc6-46d6-ae00-ba4b25313c10", "2018-09-14T133314.453337Z"))
```

Comment:
Is this necessary? This test isn't actually hitting the index.

Reply:
I thought that I needed a valid bundle to test (`-0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb-', 400). But after thinking it over, I realized the canned bundle isn't necessary.